services/configuration.go

```diff
@@ -26,7 +26,7 @@ var DefaultExplorerConfiguration types.ExplorerConfigurationMap = types.Explorer
 	ConfigurationCategorySlotViz: {
 		ConfigurationKeyVisibleFromEpoch: {Value: "0", DataType: "int"},
 		ConfigurationKeyHardforkEpoch:    {Value: "0", DataType: "int"},
-		ConfigurationKeyVisibleToEpoch:   {Value: "0", DataType: "int"},
+		ConfigurationKeyVisibleToEpoch:   {Value: "4", DataType: "int"},
```

Comment:
I'm curious: What does this do?

Reply:
It adds a more appropriate default value for new networks imo. This will cause the index page to show the slot visualizer for the first few epochs without additional effort ([see here](https://github.com/gobitfly/eth2-beaconchain-explorer/blob/master/handlers/index.go#L73-L87)). That can be useful for local testnets or forks of the project, because you don't have to manually insert the configs into the database first.

Reply:
I see, this also resolves https://github.com/gobitfly/eth2-beaconchain-explorer/pull/2590#discussion_r1369787009, nice